psl/psl-core/src/common/preview_features.rs

```diff
@@ -132,7 +132,7 @@ pub const ALL_PREVIEW_FEATURES: FeatureMap = FeatureMap {
         | TransactionApi
         | UncheckedScalarInputs
     }),
-    hidden: enumflags2::make_bitflags!(PreviewFeature::{PrismaSchemaFolder | ReactNative}),
```

Comment:
It should be added to `active` above I think?

Reply:
yep, it should